xcube_geodb/core/geodb.py

```diff
@@ -969,58 +1007,73 @@ def list_my_grants(self) -> Union[DataFrame, Message]:
         except Exception as e:
             return self._maybe_raise(GeoDBError(str(e)))
 
-    def add_property(self, collection: str, prop: str, typ: str, database: Optional[str] = None) -> Message:
+    def add_property(self, collection: str, prop: str, typ: str,
+                     database: Optional[str] = None) -> Message:
         """
         Add a property to an existing collection.
 
         Args:
             collection (str): The name of the collection to add a property to
             prop (str): Property name
             typ (str): The data type of the property (Postgres type)
-            database (str): The database the collection resides in [current database]
+            database (str): The database the collection resides in [current
+                            database]
 
         Returns:
-            Message: Success Message
+            Message: Success message
 
         Examples:
             >>> geodb = GeoDBClient()
-            >>> geodb.add_property(collection='[MyCollection]', name='[MyProperty]', type='[PostgresType]')
+            >>> geodb.add_property(collection='[MyCollection]', \
+                name='[MyProperty]', type='[PostgresType]')
         """
         prop = {prop: typ}
 
-        return self.add_properties(collection=collection, properties=prop, database=database)
+        return self.add_properties(collection=collection,
+                                   properties=prop, database=database)
 
-    @deprecated_kwarg('namespace', 'database')
-    def add_properties(self, collection: str, properties: Dict, database: Optional[str] = None, **kwargs) -> Message:
+    def add_properties(self, collection: str, properties: Dict,
+                       database: Optional[str] = None) -> Message:
         """
         Add properties to a collection.
 
         Args:
             collection (str): The name of the collection to add properties to
             properties (Dict): Property definitions as dictionary
-            database (str): The database the collection resides in [current database]
+            database (str): The database the collection resides in [current
+                            database]
         Returns:
             Message: Whether the operation succeeded
 
         Examples:
-            >>> properties = {'[MyName1]': '[PostgresType1]', '[MyName2]': '[PostgresType2]'}
+            >>> properties = {'[MyName1]': '[PostgresType1]', \
+                              '[MyName2]': '[PostgresType2]'}
             >>> geodb = GeoDBClient()
-            >>> geodb.add_property(collection='[MyCollection]', properties=properties)
+            >>> geodb.add_property(collection='[MyCollection]', \
+                                   properties=properties)
         """
 
         self._refresh_capabilities()
 
         database = database or self.database
-        collection = database + '_' + collection
+        dn = database + '_' + collection
```

Comment:
what is dn? I would prefer a more self describing variable name.

Reply:
dn = distinguished name (I guess)

Yes, I'd also prefer a better name, but I put it here for consistency reasons over the geodb client (it's named like this everywhere). Renaming is out of the scope of this PR.